poppy/tests/test_fresnel.py

```diff
@@ -196,7 +196,13 @@ def test_Circular_Aperture_PTP_short(display=False, npix=512, display_proper=Fal
         npix=npix,
         oversample=4)
     wf *= optics.CircularAperture(radius=800 * 1e-9*u.m)
-    wf_2 = wf.copy()
+    wf_2 = fresnel.FresnelWavefront(
+        2 * u.um,
+        wavelength=10e-9*u.m,
+        npix=npix,
+        oversample=4)
+    wf_2 *= optics.CircularAperture(radius=800 * 1e-9*u.m)
+
```

Comment:
Out of curiosity was there a reason you needed to duplicate this test code instead of copying the wavefront?

Reply:
wavefront doesn't deepcopy with the cuFFTPLAN plan, it can't be pickled. the plan isn't initialized unless the cuda flag is enabled but I wanted to pass the test with cuda enabled.